src/Environment.php

```diff
@@ -37,7 +38,7 @@ public function __construct(Application $app)
     {
         $this->app = $app;
 
-        if ($this->installed() && config('tenancy.hostname.auto-identification')) {
+        if (config('tenancy.hostname.auto-identification')) {
```

Comment:
Where are we calling `installed()` after you remove it from here?  Does the method still need to exist, or was it a mistake to remove the call from this line?

Reply:
Yeah, it's needed for tests, but not real http requests.